client/src/featureform/local_utils.py

```diff
@@ -0,0 +1,147 @@
+import re
+
+import pandas as pd
```

Comment:
These functions were bloating the LocalClient file so I moved them out in a utils file. Downside is we have to switch off files but upside is a leaner file with the core functionality in the file (and class)

Reply:
This is a refactor!